src/web/js/repl-ui.js

```diff
@@ -319,13 +320,13 @@
         lastEditorRun = uiOptions.cm || null;
         setWhileRunning();
 
-        editors = {};
-        editors["definitions://"] = uiOptions.cm;
-        function invalidateHighlights(cm, change) {
-          cm.off("change", invalidateHighlights);
-          document.getElementById("main").dataset.highlights = "";
-        }
-        editors["definitions://"].on("change", invalidateHighlights);
+        CPO.documents.forEach(function(doc, name) {
+          if (name.indexOf("interactions://") === 0)
```

Comment:
General comment that it would be good to define a constant for `"interactions://"` and `"definitions://"` to use in more places.  I'm guilty of this, too, and it made some changes in early summer harder to make.